geometry_msgs/PoseStamped inPose
bool isAngle
---
float64[] joint